Jenkinsfile

```diff
@@ -285,10 +265,11 @@ pipeline {
                         gitPrep()
                         crlKeyAdd()
 
-                        // Ensure that the docker tag in docker_version_tag is same as in Dockerfile.jenkins
                         script {
-                            String dockerVersionTag = readFile(file: 'scripts/docker_version_tag').trim()
-                            sh("""grep "FROM .*:${dockerVersionTag}" scripts/Dockerfile.jenkins""")
+                            // Ensure that the docker tag in docker_version_tag is same as in Dockerfile.jenkins
```

Comment:
Let's add some follow-on stories to clean-up the scripts for building docker images in the `scripts` dir of PBR?  We can delete this code, remove the `docker_version_tag` file, and the scripts that are used to build the image.

Reply:
CS-1181